### PR TITLE
fetching propertyDef from the cache memory

### PR DESCRIPTION
**Pull request by Praveen245Kumar**

### CODE REVIEW

File: copper-server-impl/src/main/java/com/pogeyan/cmis/impl/utils/DBUtils.java [hunk 1]
```diff
@@ -430,11 +429,21 @@ public static IDocumentObject getByDocumentByPropertiesField(String repositoryId
...
-		public static Map<String, PropertyDefinition<?>> getAllPropertyById(String repositoryId, String propId,
+		@SuppressWarnings("unchecked")
+		public static PropertyDefinition<?> getAllPropertyById(String repositoryId, String propId,
 				String[] fieldAccess) {
-			MTypeManagerDAO typeManagerDAO = DatabaseServiceFactory.getInstance(repositoryId)
-					.getObjectService(repositoryId, MTypeManagerDAO.class);
-			return typeManagerDAO.getAllPropertyById(propId, fieldAccess);
+			List<? extends TypeDefinition> typeDef = ((List<TypeDefinition>) CacheProviderServiceFactory
+					.getTypeCacheServiceProvider().get(repositoryId, null));
+			TypeDefinition propDef = typeDef.stream().filter(a -> a.getPropertyDefinitions().get(propId) != null)
```
Comment (fahadsuhaib) [inline copper-server-impl/src/main/java/com/pogeyan/cmis/impl/utils/DBUtils.java]: rename propDef to typeDef

File: copper-server-impl/src/main/java/com/pogeyan/cmis/impl/utils/DBUtils.java [hunk 2]
```diff
@@ -430,11 +429,21 @@ public static IDocumentObject getByDocumentByPropertiesField(String repositoryId
...
 		}
 
-		public static Map<String, PropertyDefinition<?>> getAllPropertyById(String repositoryId, String propId,
+		@SuppressWarnings("unchecked")
+		public static PropertyDefinition<?> getAllPropertyById(String repositoryId, String propId,
 				String[] fieldAccess) {
-			MTypeManagerDAO typeManagerDAO = DatabaseServiceFactory.getInstance(repositoryId)
-					.getObjectService(repositoryId, MTypeManagerDAO.class);
-			return typeManagerDAO.getAllPropertyById(propId, fieldAccess);
+			List<? extends TypeDefinition> typeDef = ((List<TypeDefinition>) CacheProviderServiceFactory
```
Comment (fahadsuhaib) [inline copper-server-impl/src/main/java/com/pogeyan/cmis/impl/utils/DBUtils.java]: rename typeDef to typeDefList

Comment (Praveen245Kumar) [reply copper-server-impl/src/main/java/com/pogeyan/cmis/impl/utils/DBUtils.java]: changed to typeDefList

File: copper-server-impl/src/main/java/com/pogeyan/cmis/impl/utils/DBUtils.java (hunk 1 above)
Comment (Praveen245Kumar) [reply copper-server-impl/src/main/java/com/pogeyan/cmis/impl/utils/DBUtils.java]: changed to typeDef

**Pull request merged**